Setup.md

```diff
@@ -159,12 +159,15 @@ We are using standard C, and CMake to build. These are the required tools:
 - [Ninja build system](https://github.com/ninja-build/ninja/releases) Version 1.10 or higher
 - GNU C++ compiler
 - SSL
+- UUID Library (if running a sample that uses correlation IDs - currently this is the Command Samples)
 
 An example of installing these tools (other than CMake) is shown below:
 
 ```bash
 sudo apt-add-repository ppa:mosquitto-dev/mosquitto-ppa
 sudo apt-get install g++-multilib ninja-build libmosquitto-dev libssl-dev
+# If running a sample that uses Correlation IDs
+sudo apt-get install uuid-dev
```

Comment:
this is only for C right ?

Reply:
yep! This is in the C section of this file - this is the library to generate the GUIDs for the correlation id